src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.EnumerateSplits.Tests.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Text.RegularExpressions.Tests
+{
+    public class RegexEnumerateSplitTests
+    {
+        [Theory]
+        [MemberData(nameof(RegexSplitTests.Split_TestData), MemberType = typeof(RegexSplitTests))]
+        public async Task Split(RegexEngine engine, string pattern, string input, RegexOptions options, int count, int start, string[] _)
+        {
+            options |= RegexOptions.ExplicitCapture; // EnumerateSplits does not include the contents of capture groups, so avoid them when possible in the test patterns.
+
+            bool isDefaultStart = RegexHelpers.IsDefaultStart(input, options, start);
+            bool isDefaultCount = RegexHelpers.IsDefaultCount(input, options, count);
+
+            Regex r = await RegexHelpers.GetRegexAsync(engine, pattern, options);
+            if (r.GetGroupNames().Length != 1)
+            {
+                // EnumerateSplits does not include the contents of capture groups.
```

Comment:
this means some tests are being skipped here that are valid tests, but just will have different expected results than for Split? is that losing coverage of any interesting cases?

Reply:
> is that losing coverage of any interesting cases?

Not really. The only ones exempt are the ones explicitly using named capture groups, and those only exist to test that the captures are included.